packages/env/README.md

```diff
@@ -1,38 +1,141 @@
+
+You can configure the port that `wp-env` uses so that it doesn't clash with another server by specifying the `WP_ENV_PORT` environment variable when starting `wp-env`:
+
+```sh
+$ WP_ENV_PORT=3333 wp-env start
+```
+
+Running `docker ps` and inspecting the `PORTS` column allows you to determine which port `wp-env` is currently using.
+
+### 3. Restart `wp-env`
+
+Restarting `wp-env` will restart the underlying Docker containers which can fix many issues.
+
+To restart `wp-env`:
+
+```sh
+$ wp-env stop
+$ wp-env start
+```
+
+### 4. Restart Docker
 
-$ cd path/to/plugin-or-theme # WordPress install will be in path/to/plugin-or-theme-wordpress.
-
-$ wp-env --help
-
-wp-env <command>
-
-Commands:
-  wp-env start [ref]                  Starts WordPress for development on port
-                                      8888 (​http://localhost:8888​) and tests
-                                      on port 8889 (​http://localhost:8889​). If
-                                      the current working directory is a plugin
-                                      and/or has e2e-tests with plugins and/or
-                                      mu-plugins, they will be mounted
-                                      appropiately.
-  wp-env stop                         Stops running WordPress for development
-                                      and tests and frees the ports.
-  wp-env clean [environment]          Cleans the WordPress databases.
-  wp-env run <container> [command..]  Runs an arbitrary command in one of the
-                                      underlying Docker containers.
-
-Options:
-  --help     Show help                                                 [boolean]
-  --version  Show version number                                       [boolean]
+Restarting Docker will restart the underlying Docker containers and volumes which can fix many issues.
+
+To restart Docker:
+
+1. Click on the Docker icon in the system tray or menu bar.
+2. Select _Restart_.
+
+Once restarted, start `wp-env` again:
+
+```sh
+$ wp-env start
 ```
 
-### `$ wp-env start --help`
+### 5. Reset the database
+
+Resetting the database which the local environment uses can fix many issues, especially when they are related to the WordPress installation.
+
+To reset the database:
+
+**⚠️ WARNING: This will permanently delete any posts, pages, media, etc. in the local WordPress installation.**
+
+```sh
+$ wp-env clean all
+$ wp-env start
+```
+
+### 6. Nuke everything and start again 🔥
+
+When all else fails, you can try forcibly removing all of the underlying Docker containers and volumes, the underlying WordPress directory, and starting again from scratch.
+
+To nuke everything:
+
+**⚠️ WARNING: This will permanently delete any posts, pages, media, etc. in the local WordPress installation.**
+
+```sh
+$ docker rm -f $(docker ps -aq)
+$ docker volume rm -f $(docker volume ls -q)
```

Comment:
> This will permanently delete any posts, pages, media, etc. **in the local WordPress installation**.

The copy should make it much clearer that these deletions will affect all possible Docker containers on the user's system, not just those concerning the immediate development environment.